### ISSUE TITLE
Connected to Xiaomi?

### ISSUE THREAD

**Issue by 1337sup3rh4x0r** [KashaMalaga/XiaomiMiBand#37]
Does the data collected from the band stay on the phone, or does this app upload it to xiaomi servers?

**Comment by CirKu17** [comment]
I recently wrote an article on [MiFit App Data Collection](https://medium.com/@cirku17/some-mifit-app-analysis-b7fbb7451059).
tl;dr it sends data to xiaomi servers and other third party companies, all unencrypted.

**Comment by 1337sup3rh4x0r** [comment]
Thank you for the article! This doesn't look good. Have you tried what happens if you block communication with these three sites? Is the app still usable? Should be, as I can also use it in flight mode, no?
Do you think it would be hard to suppress communication by modifying the app within this very project here?

**Comment by CirKu17** [comment]
Yes, it is. I run the app under the Cyanogen PrivacyGuard and I blocked the domains in the hosts file.
Maybe, that's something the maintainer should consider.
